### DISCUSSION TITLE
Commit message specification

### DISCUSSION THREAD

**Discussion by dengn** [matrixorigin/matrixone#2155] [Ideas]
MatrixOne commit message is still not standardized. We need a commit specification to standardize our commit message format and limit the commit number in a pull request. Thanks to @noneback for bringing this out with issue #2126.

This discussion is started for determining about this specification. All proposals and comments are welcome.

**Comment by noneback** [comment]
- commit message : [conventional commits](https://www.conventionalcommits.org/zh-hans/v1.0.0/)
- commits number limitation: A PR should be implemented in one commit. If not, the original PR should be divided into several PRs
